packages/core/stories/tutorials/combo-charts.ts

```diff
@@ -0,0 +1,114 @@
+	content: marked(`
+# Combo charts
+Combo charts allow users to visually compare complex datasets by superimposing different graphing components within one chart.
+Combination charts **may or may not** use dual axes although it is common that they do. For more information on dual axes with **carbon-charts**, please see [this tutorial](https://carbon-design-system.github.io/carbon-charts/?path=/story/tutorials--dual-axes-charts).
+
+*Note: \`ComboChart\` extends axis chart functionality and should only comprise of graphs/components that use axes.*
+
+____
+
+### Using the carbon-charts defaults
+
+There is support for using pre-existing carbon-charts graphs and combining them together within a \`ComboChart\`.
+The charts that can be seamlessly combined are all different **axis charts only**. This is the **reccommended way to use \`ComboCharts\`**.
+
+Supported \`ChartTypes\` to combine:
+- \`LineChart\`
+- \`ScatterChart\`
+- \`StackedScatterChart\`
+- \`AreaChart\`
+- \`StackedAreaChart\`
+- \`BarChart\`
+- \`GroupedBarChart\`
+- \`StackedBarChart\`
+
+Below is an example of combining a carbon-charts \`AreaChart\` with a \`LineChart\`.
+
+\`\`\`
+import { ChartTypes } "@carbon/charts/interfaces";
+const verySimpleComboData = [
+    { group: "Dataset 1", value: 650, date: "Tuesday" },
+	{ group: "Dataset 2", value: 296, date: "Tuesday" }
+	...more data
+];
+
+const comboOptions = {
+	...otherOptions // global options apply to all the charts listed in chartTypes (where relevant)
+	axes: {...}
+	comboChartTypes: [ // takes an array of objects
+		{
+			type: ChartTypes.AREA // use a valid ChartType from enums
+			options: {}, // component specific options (i.e, turn on the scatter points, adjust scatter radius)
+			correspondingDatasets: ["Dataset 1"] // the datasets that we want to map with this type of chart
+		},
+		{
+			type: ChartTypes.LINE // use a valid ChartType from enums
+			options: {
+				points: {
+					enabled: false; // disable the point on the LineChart won't affect the points in the AreaChart
+				}
+			}, // component specific options
+			correspondingDatasets: ["Dataset 2"]
+		}]
+};
+\`\`\`
+
+See the storybook demos for more combinations of charts and configurations.
+
+____
+
+### Using custom graphs & components
+Custom components and **independent** carbon-charts components can also be rendered into a \`ComboChart\`. This allows
+users to create their own components and combine them with the pre-existing componentry offered.
+
+Below is an example of combining a carbon-charts \`AreaChart\` with a standalone \`Line\` class constructor.
+
+**Reminder: \`ComboChart\` extends axis chart functionality and should only comprise of graphs/components that use axes.**
+
+
+\`\`\`
+// import standalone componentry
+import { Line } from "@carbon/charts/components/index";
+import { options } from "@carbon/charts/configuration"; // need to include base options for the carbon-chart if you are using a component separately
```

Comment:
shouldn't the library instead just use the default line options and merge with whatever you provide?

Reply:
yes when they want to use the enums we do (the suggested way to use combo charts), but if they are using the constructors then they should deal with it themselves? 
This is obviously for the case where the developer is making something we dont suggest/support in carbon charts so one would think they should be controlling the options also without us doing anything under the surface? thats my opinion because if youre trying to check the actual type of the constructor and then check if that is within carbon charts and then apply the base styles - then you should have used an enum. ScatterChart would have base options but on just Scatter as a graph or other smaller components ? let me know your thoughts on that or if that doesnt make sense